src/main/java/org/takes/rs/RsPrettyXML.java

```diff
@@ -114,10 +118,53 @@ private Response make() throws IOException {
                 OutputKeys.OMIT_XML_DECLARATION, "yes"
             );
             transformer.setOutputProperty(OutputKeys.INDENT, "yes");
+            if (isHtml5(source.getInputSource().getByteStream())) {
+                // @checkstyle MultipleStringLiteralsCheck (1 line)
+                transformer.setOutputProperty(OutputKeys.METHOD, "html");
+                transformer.setOutputProperty(OutputKeys.VERSION, "5.0");
+            }
             transformer.transform(source, new StreamResult(result));
         } catch (final TransformerException ex) {
             throw new IOException(ex);
         }
         return result.toByteArray();
     }
+
+    /**
+     * Checks if the input is an html5 document.
+     * @param body The body to be checked.
+     * @return True if the input is an html5 document.
+     * @checkstyle MethodNameCheck (4 lines)
+     */
+    @SuppressWarnings({ "PMD.AvoidCatchingGenericException",
```

Comment:
@carlosmiranda again because it's not a "real" exception. In both cases I just need to return false.